### PR TITLE
Add nl.brixit.wiremapper

### PR DESCRIPTION
**Pull request by MartijnBraam**

### CODE REVIEW

File: nl.brixit.wiremapper.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--share=ipc",
+    "--system-talk-name=org.bluez",
+    "--allow=bluetooth",
+    "--share=network"
+  ],
+  "modules": [
+    "python3-wiremapper.json",
+    {
```
Comment (bilelmoussaoui) [inline nl.brixit.wiremapper.json]: Leftover?

File: python3-wiremapper.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/52/e7/464079606a9cf97ad04936c52a5324d14dae36215f9319bf3faa46a7907d/scapy-2.4.3.tar.gz",
+            "sha256": "e2f8d11f6a941c14a789ae8b236b27bd634681f1b29b5e893861e284d234f6b0"
+        },
+        {
+            "type": "file",
+            "url": "https://files.pythonhosted.org/packages/62/7e/d4fb56a1695fa65da0c8d3071855fa5408447b913c58c01933c2f81a269a/dbus-python-1.2.16.tar.gz",
+            "sha256": "11238f1d86c995d8aed2e22f04a1e3779f0d70e587caffeab4857f3c662ed5a4"
+        },
+        {
```
Comment (bilelmoussaoui) [inline python3-wiremapper.json]: already in the runtime

**Comment by bilelmoussaoui** [comment]
bot, build nl.brixit.wiremapper

File: python3-wiremapper.json (hunk 2 above)
Comment (MartijnBraam) [reply python3-wiremapper.json]: no it isn't

File: nl.brixit.wiremapper.json (hunk 1 above)
Comment (MartijnBraam) [reply nl.brixit.wiremapper.json]: no

File: python3-wiremapper.json (hunk 2 above)
Comment (bilelmoussaoui) [reply python3-wiremapper.json]: just checked, indeed it isn't there anymore

File: nl.brixit.wiremapper.json (hunk 1 above)
Comment (MartijnBraam) [reply nl.brixit.wiremapper.json]: Well actually, I have no idea what it exactly does. But all examples I've found have this.

Comment (bilelmoussaoui) [reply nl.brixit.wiremapper.json]: I was talking about the empty module wiremapper that does nothing?

File: nl.brixit.wiremapper.json [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+          "url": "https://source.puri.sm/Librem5/libhandy.git",
+          "tag": "v0.0.13",
+          "commit": "7a193d7692c9c76a1a94f17c4d30b585f77d177c"
+        }
+      ]
+    },
+    {
+      "name": "desktop-files",
+      "buildsystem": "simple",
+      "build-commands": [
```
Comment (bilelmoussaoui) [inline nl.brixit.wiremapper.json]: Any reason for not installing the wiremapper package here and using the source from pip? that's what most apps do and they generate the sources from a requirements.txt file or something equivalent for the app dependencies

Comment (bilelmoussaoui) [reply nl.brixit.wiremapper.json]: This should also allow you to use the manifest with GNOME Builder/Gitlab CI

File: nl.brixit.wiremapper.json (hunk 1 above)
Comment (MartijnBraam) [reply nl.brixit.wiremapper.json]: yeah that was in all the examples I could find that included another json file, an empty block with only a name after it. I'll remove it.

File: nl.brixit.wiremapper.json (hunk 3 above)
Comment (MartijnBraam) [reply nl.brixit.wiremapper.json]: I like to keep my non-python-module stuff seperate from it since installing that stuff using pip is a bit of a hack.

Comment (bilelmoussaoui) [reply nl.brixit.wiremapper.json]: Well, in such case, just use a file source for the metainfo/desktop/icon to not clone the whole repository

**Comment by MartijnBraam** [comment]
Is it up to standard now?

**Comment by bilelmoussaoui** [comment]
> Is it up to standard now?

Sorry, I wasn't clear with the usage of file source. The files should live upstream, just use `url` instead of `path` and point it the files upstream. See https://github.com/flathub/flathub/pull/1418/files#diff-cc6ecc8efbc2ac718fe421f3356eac94R48 for an example

**Comment by bilelmoussaoui** [comment]
Other than this, it looks ready to me

**Comment by MartijnBraam** [comment]
Why? the metainfo should probably be updated at the same time as the flatpak metadata, this seems to make a lot of  sense actually.

**Comment by bilelmoussaoui** [comment]
These files are not flatpak specifics

**Comment by MartijnBraam** [comment]
done

**Comment by bilelmoussaoui** [comment]
bot, build nl.brixit.wiremapper

**Pull request closed without merging**